### DISCUSSION TITLE
essay/2024/04/04/online-coffee-chat

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#203] [Comments]
# essay/2024/04/04/online-coffee-chat

온라인으로 누군가와 얼굴 보고 대화는 나누는 건 어색한 일이었습니다.

https://jeho.page/essay/2024/04/04/online-coffee-chat.html

**Comment by bookseal** [comment]
기쁜 마음이 전달되는 잘 쓴 글이네요
